Index: linux-custom/arch/arm/boot/dts/Makefile
===================================================================
--- linux-custom.orig/arch/arm/boot/dts/Makefile
+++ linux-custom/arch/arm/boot/dts/Makefile
@@ -801,6 +801,7 @@ dtb-$(CONFIG_SOC_AM33XX) += \
 	am335x-base0033.dtb \
 	am335x-bone.dtb \
 	am335x-boneblack.dtb \
+	am335x-boneblack-bbbmini.dtb \
 	am335x-sancloud-bbe-extended-wifi-uboot-univ.dtb \
 	am335x-sancloud-bbe-lite-uboot-univ.dtb \
 	am335x-sancloud-bbe-uboot-univ.dtb \
